From e19dda32f4a2a8186c36259d49473be1eddfd031 Mon Sep 17 00:00:00 2001
From: Jack Olivieri <jolivieri@anaconda.com>
Date: Thu, 2 May 2024 21:36:43 +0200
Subject: [PATCH 5/6] fix broken text filters

---
 oss_scripts/configure.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/oss_scripts/configure.sh b/oss_scripts/configure.sh
index 8057139..f9b3032 100755
--- a/oss_scripts/configure.sh
+++ b/oss_scripts/configure.sh
@@ -83,8 +83,8 @@ TF_LFLAGS=( $($installed_python -c "import tensorflow as tf; print(' '.join(tf.s
 TF_LFLAGS_2=( $($installed_python -c "import tensorflow as tf; print(' '.join(tf.sysconfig.get_link_flags()))" | awk '{print $2}') )
 TF_ABIFLAG=$($installed_python -c "import tensorflow as tf; print(tf.sysconfig.CXX11_ABI_FLAG)")
 
-HEADER_DIR=${TF_CFLAGS:2}
-SHARED_LIBRARY_DIR=${TF_LFLAGS:2}
+HEADER_DIR=$(echo $TF_CFLAGS | cut -c 3-)
+SHARED_LIBRARY_DIR=$(echo $TF_LFLAGS | cut -c 3-)
 SHARED_LIBRARY_NAME=$(echo $TF_LFLAGS_2 | rev | cut -d":" -f1 | rev)
 if is_macos; then
   SHARED_LIBRARY_NAME="libtensorflow_framework.2.dylib"
-- 
2.39.3 (Apple Git-146)

