From 5a676cce5119c0f7d4d5f29a276f9d89bfa26523 Mon Sep 17 00:00:00 2001
From: Jack Olivieri <jolivieri@anaconda.com>
Date: Mon, 6 May 2024 19:44:19 +0200
Subject: [PATCH] stat file to see if it is missin on prefect log

---
 oss_scripts/configure.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/oss_scripts/configure.sh b/oss_scripts/configure.sh
index f9b3032..2da5a42 100755
--- a/oss_scripts/configure.sh
+++ b/oss_scripts/configure.sh
@@ -99,3 +99,6 @@ write_action_env_to_bazelrc "TF_HEADER_DIR" ${HEADER_DIR}
 write_action_env_to_bazelrc "TF_SHARED_LIBRARY_DIR" ${SHARED_LIBRARY_DIR}
 write_action_env_to_bazelrc "TF_SHARED_LIBRARY_NAME" ${SHARED_LIBRARY_NAME}
 write_action_env_to_bazelrc "TF_CXX11_ABI_FLAG" ${TF_ABIFLAG}
+
+echo "stat $\{SHARED_LIBRARY_DIR}/$\{SHARED_LIBRARY_NAME} 2>&1"
+stat ${SHARED_LIBRARY_DIR}/${SHARED_LIBRARY_NAME} 2>&1
-- 
2.39.3 (Apple Git-146)

